### DISCUSSION TITLE
This has got me excited about AI again

### DISCUSSION THREAD

**Discussion by stavrostzagadouris** [danielmiessler/Fabric#58] [General]
Hey @danielmiessler, I've been a newsletter subscriber for a while and just watched your video where you showed this tool to David Bombal. I like your coding interface. This is so well thought out and executed.

Anyway I just wanted to say, I've been playing with this stuff for a year, making my openai python Discord bot more and more feature packed and then I kind of hit a wall.
I have it summarize youtubes, or articles if you give it a url. 
It connects to my comfyui desktop to make pics. I have a version connected to openai and another copy/paste bot with a different name connected to my own lm-studio llm's to play with them.
My most recent silly idea is 'braintrust' where you ask it a question and it makes up 4 different professionals with names to answer your question from their point of view. Not really more useful than just asking a question, but more fun for sure.
My bots have been SO useful to me, I use it every day and really notice if it goes down for whatever reason. I can just consume SO much more and so faster, it feels like I'm cheating at life.

Seeing what you've done, and especially perusing your patterns has got me excited to implement your wicked (and markdown! cool idea) prompts into my bot as even more tools.

I've already replaced my summarizer with yours and it's just heaps better.

You got me enthused all over again and I wanted to share that with you.

I'm also interested in getting fabric running on my computer proper as well, but I wouldn't want to replace my discord bots fully as they give me mobile access anywhere I have discord. Do you have plans to make yours available remotely via app somehow? I'd love to but never taken the first step.

Thanks for the wicked content!

**Comment by cbruyndoncx** [comment]
I also arrived here because of the video.

**Comment by danielmiessler** [comment]
Wow, very cool. Thank you so much for the feedback. And this is the project at 2% of its potential. Imagine what it'll be in 6 months!
